oct/cli/package/aws.py

```diff
@@ -9,6 +9,11 @@
 _SHORT_HELP = 'Package a running AWS EC2 virtual machine.'
 
 
+class Tag(object):
```

Comment:
It's Python so I'm not really sure you need this -- just pass a `dict` into the playbook